### PR TITLE
IPA fixes

### PR DESCRIPTION
**Pull request by mwgamera**
I did some systematic check trying to verify if Shavian spellings can be generated from the IPA field.  I skipped entries where IPA is missing, abbreviations, and acronyms.  I ignored entries which differed only by having ‹𐑦𐑩› (or ‹𐑰𐑩›) where I expected ‹𐑾›.  I also ignored these where I weren't sure how to fix the discrepancy, i.e.:

* arowana — ‹𐑪› or ‹𐑭›?
* Audley — IPA field says it should end with the happy vowel but the only source I know that gives the pronunciation of it ([CUBE](http://cube.elte.hu/index.pl?s=audley&gimson=on)) equates happy with FLEECE;
* casuist casuistic casuistical casuistries casuistry casuists — I can't figure which variant is right for RP, but IPA is certainly wrong;
* countess countesses deaconess deaconesses — Shavian with ‹𐑧› /e/ seems right, but apparently pronunciation with /ə/ or even /ɪ/ is possible so maybe it should be ‹𐑩› /I/ after all?
* employé — I think this should be the same as ‹employee› *in English*, but if it tries to approximate French then it has wrong vowels and is missing ‹𐑘› (cf. employés);
* jukka — ‹𐑭› or ‹𐑩›?
* spiral spiralled spiralling spirally spirals spirals — IPA has one ‹ə› too much in comparison with Shavian, but it's a possible pronunciation too;
* Tremens — if it's ‹delirium tremens› then it's the same as with ‘countess’, but it's marked as a proper name and I couldn't find what it refers to;
* Verma — Shavian is probably missing the vowel at the end, but I couldn't find how it should be pronounced.

All the others either looked correct, or I fixed them below.  Changes include some obvious errors that should be uncontroversial to fix, but also less obvious changes like those in punctuation to match the spelling (spaces vs hyphens), or a class of words with ‹eəRˈr› which is an impossible sequence that would suggest ‹𐑺𐑮› spelling.  I also corrected some ‹r› vs ‹R› which don't influence Shavian spelling at all.  But a careful review is needed in any case. Thanks.

### CODE REVIEW

File: kingsleyreadlexicon.tsv [hunk 1]
```diff
@@ -4250,11 +4250,11 @@ ashford	𐑨𐑖𐑓𐑼𐑛	NP0	ˈæʃ.fəRd	115
...
 ashore	𐑩𐑖𐑹	AV0	əˈʃɔːR	647
 ashraf	𐑨𐑖𐑮𐑨𐑓	NP0	ˈæʃræf	23
-ashton-in-makerfield	𐑨𐑖𐑑𐑩𐑯-𐑦𐑯-𐑥𐑱𐑒𐑼𐑓𐑰𐑤𐑛	NP0	ˌæʃ.tən ɪn ˈmeɪ.kəR.fiːld	1
-ashton-under-lyne	𐑨𐑖𐑑𐑩𐑯-𐑳𐑯𐑛𐑼-𐑤𐑲𐑯	NP0	ˌæʃ.tən ˌʌn.dəR ˈlaɪn	30
+ashton-in-makerfield	𐑨𐑖𐑑𐑩𐑯-𐑦𐑯-𐑥𐑱𐑒𐑼𐑓𐑰𐑤𐑛	NP0	ˌæʃ.tən-ɪn-ˈmeɪ.kəR.fiːld	1
+ashton-under-lyne	𐑨𐑖𐑑𐑩𐑯-𐑳𐑯𐑛𐑼-𐑤𐑲𐑯	NP0	ˌæʃ.tən-ˌʌn.dəR-ˈlaɪn	30
 ashtray	𐑨𐑖𐑑𐑮𐑱	NN1	æʃtreɪ	210
 ashtrays	𐑨𐑖𐑑𐑮𐑱𐑟	NN2	æʃtreɪz	68
-ashworth	𐑨𐑖𐑢𐑻𐑔	NP0	ˈæʃ.wəRθ	148
+ashworth	𐑨𐑖𐑢𐑻𐑔	NP0	ˈæʃ.wɜːRθ	148
```
Comment (Shavian-info) [inline kingsleyreadlexicon.tsv:4257]: To follow the first listed pronunciation in the Cambridge Pronouncing Dictionary I suggest:

`ashworth	𐑨𐑖𐑢𐑼𐑔	NP0	ˈæʃ.wəːRθ	148`

File: kingsleyreadlexicon.tsv [hunk 2]
```diff
@@ -4651,7 +4651,7 @@ athenian	𐑩𐑔𐑰𐑯𐑾𐑯	AJ0	əˈθiːnɪən	295
 athenian	𐑩𐑔𐑰𐑯𐑾𐑯	NN1	əˈθiːnɪən	1
 athenians	𐑩𐑔𐑰𐑯𐑾𐑯𐑟	NN2	əˈθiːnɪənz	148
 athens	𐑨𐑔𐑩𐑯𐑟	NP0	ˈæθ.ənz	1039
-atherstone	𐑨𐑔𐑼𐑕𐑑𐑩𐑯	NP0	ˈæθ.əR.stəʊn	14
+atherstone	𐑨𐑔𐑼𐑕𐑑𐑩𐑯	NP0	ˈæθ.əR.stən	14
```
Comment (Shavian-info) [inline kingsleyreadlexicon.tsv:4654]: To follow the first listed pronunciation in the Cambridge Pronouncing Dictionary I suggest:

`atherstone	𐑨𐑔𐑼𐑕𐑑𐑴𐑯	NP0	ˈæθ.əR.stəʊn	14`

File: kingsleyreadlexicon.tsv [hunk 3]
```diff
@@ -56256,8 +56256,8 @@ mobcaps	𐑥𐑪𐑚𐑒𐑨𐑐𐑕	NN2	mɒbkæps	0
 mobile	𐑥𐑴𐑚𐑲𐑤	AJ0	məʊbaɪl	923
 mobile	𐑥𐑴𐑚𐑲𐑤	NN1	məʊbaɪl	65
 mobiles	𐑥𐑴𐑚𐑲𐑤𐑟	NN2	məʊbaɪlz	27
-mobilisation	𐑥𐑴𐑚𐑩𐑤𐑲𐑟𐑱𐑖𐑩𐑯	NN1	ˌməʊbɪlaɪˈzeɪʃən	132
-mobilisations	𐑥𐑴𐑚𐑩𐑤𐑲𐑟𐑱𐑖𐑩𐑯𐑟	NN2	ˌməʊbɪlaɪˈzeɪʃənz	2
+mobilisation	𐑥𐑴𐑚𐑩𐑤𐑲𐑟𐑱𐑖𐑩𐑯	NN1	ˌməʊbIlaɪˈzeɪʃən	132
+mobilisations	𐑥𐑴𐑚𐑩𐑤𐑲𐑟𐑱𐑖𐑩𐑯𐑟	NN2	ˌməʊbIlaɪˈzeɪʃənz	2
```
Comment (Shavian-info) [inline kingsleyreadlexicon.tsv:56260]: I suggest these should be the following for consistency:

```
mobilisation	𐑥𐑴𐑚𐑩𐑤𐑲𐑟𐑱𐑖𐑩𐑯	NN1	ˌməʊbəlaɪˈzeɪʃən	132
mobilisations	𐑥𐑴𐑚𐑩𐑤𐑲𐑟𐑱𐑖𐑩𐑯𐑟	NN2	ˌməʊbəlaɪˈzeɪʃənz	2
```

File: kingsleyreadlexicon.tsv [hunk 4]
```diff
@@ -82208,9 +82208,9 @@ simulations	𐑕𐑦𐑥𐑘𐑩𐑤𐑱𐑖𐑩𐑯𐑟	NN2	ˌsɪmjəˈleɪʃə
 simulator	𐑕𐑦𐑥𐑘𐑩𐑤𐑱𐑑𐑼	NN1	sɪmjəleɪtəR	116
 simulators	𐑕𐑦𐑥𐑘𐑩𐑤𐑱𐑑𐑼𐑟	NN2	sɪmjəleɪtəRz	53
 simultaneity	𐑕𐑦𐑥𐑩𐑤𐑑𐑩𐑯𐑱𐑦𐑑𐑦	NN1	ˌsɪməltəˈneɪɪti	21
-simultaneous	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕	AJ0	ˌsɪmlˈteɪnɪəs	642
-simultaneously	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕𐑤𐑦	AV0	ˌsɪmlˈteɪnɪəsli	1770
-simultaneousness	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕𐑯𐑩𐑕	NN1	ˌsɪmlˈteɪnɪəsnəs	0
+simultaneous	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕	AJ0	ˌsɪmƏlˈteɪnɪəs	642
+simultaneously	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕𐑤𐑦	AV0	ˌsɪmƏlˈteɪnɪəsli	1770
+simultaneousness	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕𐑯𐑩𐑕	NN1	ˌsɪmƏlˈteɪnɪəsnəs	0
```
Comment (Shavian-info) [inline kingsleyreadlexicon.tsv:82213]: I suggest this should be the following for consistency:

```
simultaneous	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕	AJ0	ˌsɪməlˈteɪnɪəs	642
simultaneously	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕𐑤𐑦	AV0	ˌsɪməlˈteɪnɪəsli	1770
simultaneousness	𐑕𐑦𐑥𐑩𐑤𐑑𐑱𐑯𐑾𐑕𐑯𐑩𐑕	NN1	ˌsɪməlˈteɪnɪəsnəs	0
```

File: kingsleyreadlexicon.tsv [hunk 5]
```diff
@@ -87912,8 +87912,8 @@ streets	𐑕𐑑𐑮𐑰𐑑𐑕	NN2	striːts	4941
 streetwalker	𐑕𐑑𐑮𐑰𐑑𐑢𐑷𐑒𐑼	NN1	striːtwɔːkəR	3
 streetwalkers	𐑕𐑑𐑮𐑰𐑑𐑢𐑷𐑒𐑼𐑟	NN2	striːtwɔːkəRz	4
 strength	𐑕𐑑𐑮𐑧𐑙𐑔	NN1	streŋθ	7077
-strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVB	streŋθn	171
-strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVI	streŋθn	1032
+strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVB	streŋθƏn	171
+strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVI	streŋθƏn	1032
```
Comment (Shavian-info) [inline kingsleyreadlexicon.tsv:87916]: I suggest these should be the following for consistency, as I don't think there's any suggestion that RP speakers might say /streŋθɪn/:

```
strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVB	streŋθən	171
strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVI	streŋθən	1032
```

**Review by Shavian-info — CHANGES_REQUESTED** [review]
Thanks so much for this! I have made a small number of requested changes, but otherwise this is fantastic.

**Comment by Shavian-info** [comment]
> * arowana — ‹𐑪› or ‹𐑭›?

This should be 𐑭. I missed this in reviewing the suggested spelling.

> * Audley — IPA field says it should end with the happy vowel but the only source I know that gives the pronunciation of it ([CUBE](http://cube.elte.hu/index.pl?s=audley&gimson=on)) equates happy with FLEECE;

This should be the HAPPY vowel. CUBE is not a reliable source on this point as it does not include the HAPPY vowel in any spellings. The authors (with good reason) argue that the HAPPY vowel is a fiction from a time when the pronunciation was transitioning from /ɪ/ to /iː/. Their argument is a good one, but I am not yet ready to take this leap with the ReadLex, on the basis that the final -𐑦/-𐑰 distinction still helps with determining stress. But who knows, maybe one day...

> * casuist casuistic casuistical casuistries casuistry casuists — I can't figure which variant is right for RP, but IPA is certainly wrong;

I think the IPA might be correct and the Shavian should be updated. Both /ˈkæz.ju.ɪst/ and /ˈkæ.ʒu.ɪst/ are given for RP in the Cambridge Pronouncing Dictionary, in that order. However, as the US pronunciation is only given as /ˈkæ.ʒu.ɪst/, following the spelling principles used for the ReadLex that pronunciation should be favoured. So the Shavian should be 𐑒𐑨𐑠𐑫𐑦𐑕𐑑.

> * countess countesses deaconess deaconesses — Shavian with ‹𐑧› /e/ seems right, but apparently pronunciation with /ə/ or even /ɪ/ is possible so maybe it should be ‹𐑩› /I/ after all?

I have gone with the stronger pronunciation of '-ess' where it is an option in natural speech. Read suggested all -ess should be -𐑩𐑕, but that would give us the less usual 𐑐𐑮𐑦𐑯𐑕𐑩𐑕 for 'princess', and create more homophones like 'adulteress' and 'adulterous'. The -𐑩𐑕 version is used where -𐑧𐑕 never appears in natural speech, e.g. 'actress'.

> * employé — I think this should be the same as ‹employee› _in English_, but if it tries to approximate French then it has wrong vowels and is missing ‹𐑘› (cf. employés);

I followed the Cambridge Pronouncing Dictionary for this one which gives /ɑ̃ːm.plɔɪˈeɪ/ or /ɔ̃ːm.plɔɪˈeɪ/. Frenchified pronunciations are notoriously difficult. I could go with 𐑭𐑥𐑐𐑤𐑶𐑱, maybe.

> * jukka — ‹𐑭› or ‹𐑩›?

It's a Finnish first name and not naturalised into English, so I went with a more Finnish pronunciation. I should really move it to the Addendum.dict.

> * spiral spiralled spiralling spirally spirals spirals — IPA has one ‹ə› too much in comparison with Shavian, but it's a possible pronunciation too;

I think we should settle on /ˈspaɪ.rəl/, even if Cambridge gives /ˈspaɪə.rəl/. This is one of the very few -aɪ(ə)r- words where it doesn't mark the first schwa as optional in RP which I think might be an error.

> * Tremens — if it's ‹delirium tremens› then it's the same as with ‘countess’, but it's marked as a proper name and I couldn't find what it refers to;

Cambridge gives /dɪˌlɪr.i.əmˈtriː.menz/, /-ˈtrem.enz/, /-ənz/. I went with the first listed which possibly reflects that this is a Latin term and not fully naturalised. But yes, it should be NN1.

> * Verma — Shavian is probably missing the vowel at the end, but I couldn't find how it should be pronounced.

I can't remember where this came from, but yes it should end in a schwa.

File: kingsleyreadlexicon.tsv (hunk 1 above)
Comment (mwgamera) [reply kingsleyreadlexicon.tsv:4257]: I think this should be ‹ə› then, not ‹əː›.

**Comment by mwgamera** [comment]
Updated with the changes requested and with those originally missed based on your feedback (except jukka).
Ashworth with ‹𐑼› means retaining /ˈæʃ.wəRθ/ as it was before (the /əː/ isn't used in this system and it denotes the NURSE vowel in systems where it is). Others verbatim.
Not being sure how desirable is ‘maybe’, I just removed the spurious /j/ from IPA of the employé as this reconciles it with the spelling already in the file and matches the latter of the pronunciations you quoted from Cambridge.

**Pull request merged**